plugins/action_generate_review_assignment_records/src/generateReviewAssignments.test.ts

```diff
@@ -61,14 +61,14 @@ describe('Submit new application on first stage to generate reviewAssignments on
       parameters: { applicationId: 1000 },
       DBConnect,
     }).then((result: any) => {
-      expect(result).toEqual({
+      expect(clearResult(result)).toEqual({
         status: ActionQueueStatus.Success,
         error_log: '',
         output: {
           reviewAssignments: [
             {
               reviewerId: 2,
-              orgId: null,
+              organisationId: null,
```

Comment:
Hmmm, when did this change from `orgId` to `organisationId`? Pretty much all the action plugins use `orgId` for their input and output parameter names (we had a discussion about it ages ago) -- wondering why this one is different?

Reply:
This is only something store locally on the `generateReviewAssignment` function, using the type defined in that plugin
![Screen Shot 2021-08-06 at 10 31 04 AM](https://user-images.githubusercontent.com/16461988/128429668-3016dae0-ed13-40f0-9fb2-b950e52f0d2a.png)